src/containers/PageBuilder/Primitives/Heading/Heading.module.css

```diff
@@ -6,24 +6,93 @@
 .h5,
 .h6 {
   margin-top: 0;
+  margin-bottom: 0;
   line-height: 1.33;
   font-weight: bold;
+  color: var(--matterColorAlmostDark);
 }
 
 /* Specific styles */
 .h1 {
   font-size: 40px;
+
+  /* If ´& + *´ if used, margin-top needs !important to overwrite */
+  & + p,
+  & + a,
+  & + ul,
+  & + ol,
+  & + code,
+  & + div,
+  & + h1,
+  & + h2,
+  & + h3,
+  & + h4,
+  & + h5,
+  & + h6 {
+    margin-top: 24px;
+  }
 }
 
 .h2 {
   font-size: 30px;
-  margin-bottom: 19px;
+
+  /* If ´& + *´ if used, margin-top needs !important to overwrite */
+  & + p,
+  & + a,
+  & + ul,
+  & + ol,
+  & + code,
+  & + div,
+  & + h1,
+  & + h2,
+  & + h3,
+  & + h4,
+  & + h5,
+  & + h6 {
```

Comment:
In what situation, the margin-top is not used?

Could we turn this around? `margin-top: 16px;` is the default and it's not there in certain situations.